crates/runtime/src/accelerated_table/refresh.rs

```diff
@@ -69,12 +71,13 @@ pub struct Refresh {
 }
 
 impl Refresh {
-    #[allow(clippy::needless_pass_by_value)]
+    #[allow(clippy::needless_pass_by_value, clippy::too_many_arguments)]
```

Comment:
I think we should take this opportunity to turn this into a builder pattern given most of the arguments are optional. i.e. `Refresh::new(RefreshMode::Full).with_max_jitter(max_jitter)`